lib/kasket/visitor.rb

```diff
@@ -119,11 +119,15 @@ def literal(node, *_)
       end
     end
 
-    def visit_Arel_Nodes_BindParam(_x, *_)
+    def visit_Arel_Nodes_BindParam(node, *_)
       if ActiveRecord::VERSION::MAJOR < 5
         visit(@binds.shift[1])
       else
-        visit(@binds.shift)
+        if ActiveRecord::VERSION::STRING < '5.2'
+          visit(@binds.shift)
+        else
+          visit(node.value.value) unless node.value.value.nil?
```

Comment:
This implicitly couples to the structure of the AST. Is it worth an intermediate visitor to avoid the chaining?